twigshim.module

```diff
@@ -48,15 +48,15 @@ function twigshim_render($template, $variables = array()) {
       if (file_exists($autoload = "$module_path/vendor/autoload.php")) {
         require_once $autoload;
       }
-      // Try the Libraries API.
-      elseif ($twig_path = libraries_get_path('twig')) {
+      // Try the Libraries API (optional depedency).
+      elseif (function_exists('libraries_get_path') && $twig_path = libraries_get_path('twig')) {
         require_once "$twig_path/lib/Twig/Autoloader.php";
         Twig_Autoloader::register();
       }
       // No Twig library was found.
       else {
-        $message = t('Please install Twig! See @link.', array(
-          '@link' => '<a href="https://github.com/derekderaps/twigshim">github.com/derekderaps/twigshim</a>',
+        $message = t('Please install Twig! See the !link for more information.', array(
+          '!link' => l(t('Twigshim Documentation'), 'https://github.com/derekderaps/twigshim')
```

Comment:
Per [d.o 322774](https://www.drupal.org/node/322774), links in translatable strings should be in the form:
```
t('Here is a <a href="@url">link</a>', array('@url' => 'http://google.com'));
```